lib/omnibus/fetchers/git_fetcher.rb

```diff
@@ -203,7 +203,7 @@ def contains_revision?(rev)
     #   the shellout object
     #
     def git(command)
-      shellout!("git #{command}", cwd: project_dir)
+      shellout!("git -c core.autocrlf=false #{command}", cwd: project_dir)
```

Comment:
@scotthain omnibus feels like the right place because it's a requirement to build some things. Like, it would feel weird if the development environment had to have the right configuration when we already know what we need and can pass it.

I can imagine us doing it only on Windows, but it feels right to do it on all ... cc @ksubrama